js/kucoin.js

```diff
@@ -1419,6 +1420,145 @@ module.exports = class kucoin extends Exchange {
         return await this[method] (this.extend (request, params));
     }
 
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
@sc0Vu since by default it only returns closed orders we should split this into two different methods, `fetchOpenOrders` and `fetchClosedOrders`because we can't get both simultaneously

Reply:
@sc0Vu can you check this comment pls?

Reply:
@carlosmiei yes, they only support one of those status, and we did implement `fetchOpenOrders` and `fetchClosedOrders`. I think we can close this PR or wait until kucoin update the parameters.